### PR TITLE
AI fix territory remove bug

### PR DESCRIPTION
**Pull request by ron-murhammer**
Fix remove to take in correct object (ProTerritory instead of Territory) so it actually removes territories that shouldn't be attacked.

---
This change is [Reviewable](https://reviewable.io/reviews/triplea-game/triplea/909)

### CODE REVIEW

File: src/games/strategy/triplea/ai/proAI/data/ProTerritoryManager.java [hunk 1]
```diff
@@ -1103,7 +1103,7 @@ private void findBombardOptions(final PlayerID player, final List<Territory> myU
       combinedUnits.addAll(patd.getMaxAmphibUnits());
       ProLogger.debug("Removing territory that we can't successfully attack: " + t + ", maxWin%="
           + patd.getMaxBattleResult().getWinPercentage() + ", maxAttackers=" + combinedUnits.size());
-      result.remove(t);
+      result.remove(attackMap.get(t));
```
Comment (DanVanAtta) [inline src/games/strategy/triplea/ai/proAI/data/ProTerritoryManager.java]: Re-use this variable: `final ProTerritory patd = attackMap.get(t);`? 
If so, would also be good to give the variable a more meaningful name as well.

**Comment by DanVanAtta** [comment]
Would you mind running a global formatting after this is merged Ron? We *really* should not have to be constantly reformatting classes, it all should be formatted already. If not, let's get it back into that state so we don't have to reformat for every PR.

**Comment by ron-murhammer** [comment]
@DanVanAtta Updated. Agree I was trying to wait til we cleared the PR queue to avoid lots of conflicts when reformatting everything.

**Comment by DanVanAtta** [comment]
There is only one AI PR open, we could create a branch off of that, and do a PR with AI code formatting changes into the one existing branch. When both PRs are approved, we merge the formatting into the open PR, and then merge the open PR into master.

**Comment by ron-murhammer** [comment]
@DanVanAtta Could. I was kind of thinking the opposite of just getting the final AI PR merged in the next few days then I'll just do a quick PR of formatting the AI code before starting any other changes.

**Pull request merged**